app/controllers/bike_share_app.rb

```diff
@@ -128,4 +126,9 @@ class BikeShareApp < Sinatra::Base
     @condition = Condition.destroy(params[:id])
     redirect "/conditions"
   end
+
+  get '/conditions-dashboard' do
```

Comment:
did we say the route should be '/conditions-dashboard' or '/conditions_dashboard'